src/patternfly/components/Tabs/tabs.scss

```diff
@@ -87,8 +89,12 @@ $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
   overflow: hidden;
 
   &::before {
+    position: absolute;
+    right: 0;
+    bottom: 0;
+    left: 0;
     border-color: var(--pf-c-tabs--before--BorderColor);
```

Comment:
can't you write this as `border: var(--pf-c-tabs--before--BorderColor) solid` and then still have your border-width declaration